types/dynamic-time-warping/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for dynamic-time-warping 1.0
+// Project: https://github.com/GordonLesti/dynamic-time-warping
+// Definitions by: Zlatko Andonovski <https://github.com/Goldsmith42>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class DynamicTimeWarping<T> {
+    constructor(ser1: ReadonlyArray<T>, ser2: ReadonlyArray<T>, distFunc: (a: T, b: T) => number);
+
+    getDistance(): number;
+    getPath(): Array<[number, number]>;
+}
+
+export = DynamicTimeWarping;
```

Comment:
This needs to be defined as UMD since https://github.com/GordonLesti/dynamic-time-warping#install mentions usage as global.
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.